### PR TITLE
Kql/title amendments

### PR DESCRIPTION
**Pull request by kevinkq**
## Summary of the changes / Why this improves CrateDB

Changes directive headings from titles case to sentence case.

## Checklist

~~- [ ] User relevant changes are recorded in ``CHANGES.txt``~~
~~- [ ] Touched code is covered by tests~~
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes

### CODE REVIEW

File: blackbox/docs/admin/ssl.rst [hunk 1]
```diff
@@ -187,20 +187,20 @@ information including the different SSL modes please visit the
 .. _psql documentation: https://www.postgresql.org/docs/current/static/app-psql.html
 
 
-Setting up a Keystore/Truststore With a Certificate Chain
+Setting up a Keystore/Truststore with a certificate chain
 =========================================================
 
-In case you need to setup a Keystore or a Trustore, here are the commands
+In case you need to setup a Kestore or a Trustore, here are the commands
```
Comment (mechanomi) [inline blackbox/docs/admin/ssl.rst]: `Kestore` looks like a typo here

File: blackbox/docs/admin/ssl.rst [hunk 2]
```diff
@@ -187,20 +187,20 @@ information including the different SSL modes please visit the
...
 
 
 .. _ssl_generate_keystore:
 
-Generate Keystore With a Private Key
+Generate Keystore with a private key
 ------------------------------------
 
-The first step is to create a keystore with a private key using the RSA
+The first step is to create a Kestore with a private key using the RSA
```
Comment (mechanomi) [inline blackbox/docs/admin/ssl.rst]: another typo

File: blackbox/docs/admin/ssl.rst [hunk 3]
```diff
@@ -342,10 +342,10 @@ Output::
 
 .. _subjectAltName: http://wiki.cacert.org/FAQ/subjectAltName
 
-Import the CA Certificate Into the Keystore
+Import the CA certificate into the Keystore
 ...........................................
 
-The CA needs to be imported to the Keystore for the certificate chain to be
+The CA needs to be imported to the Kestore for the certificate chain to be
```
Comment (mechanomi) [inline blackbox/docs/admin/ssl.rst]: typo

File: blackbox/docs/admin/ssl.rst [hunk 4]
```diff
@@ -452,11 +452,11 @@ Output::
     Certificate was added to keystore
 
 
-Import the Signed Certificate
+Import the signed certificate
 -----------------------------
 
 Now we have a signed certificate, signed by either from a official CA
-or from our own CA. Let's import it to the keystore.
+or from our own CA. Let's import it to the Kestore.
```
Comment (mechanomi) [inline blackbox/docs/admin/ssl.rst]: typo

File: blackbox/docs/config/cluster.rst [hunk 5]
```diff
@@ -1098,3 +1098,37 @@ Metadata gateway
   because you only want the cluster state to be recovered once all nodes are
   started. However, the value must be bigger than the half of the expected
   number of nodes in the cluster.
+
+.. _s3-credentials:
```
Comment (kovrus) [inline blackbox/docs/config/cluster.rst]: Is there a reason why it was brought back? Those settings cannot be via crate.yaml any longer.

File: blackbox/docs/sql/statements/create-repository.rst [hunk 6]
```diff
@@ -357,6 +357,12 @@ A repository type that stores its snapshots on the Azure Storage service.
   The location mode for storing data on the Azure Storage.
   Note that if you set it to ``secondary_only``, it will force readonly to true.
 
+Azure client settings
+.....................
+
+All the setting values are specified via ``azure.client.`` in the crate.yaml
```
Comment (kovrus) [inline blackbox/docs/sql/statements/create-repository.rst]: Is there a reason why this section was introduced? It is also not correct.

File: blackbox/docs/admin/snapshots.rst [hunk 7]
```diff
@@ -48,6 +48,8 @@ Creating a repository with the same name will result in an error::
     ... WITH (location='another_repo_path', compress=false);
     SQLActionException[RepositoryAlreadyExistsException: Repository 'where_my_snapshots_go' already exists]
 
+.. _plugins: https://github.com/crate/crate/blob/master/devs/docs/plugins.rst
```
Comment (kovrus) [inline blackbox/docs/admin/snapshots.rst]: by referencing `https://github.com/crate/crate/blob/master/devs/docs/plugins.rst` you will aways reference plugin docs from master, but not the current version of docs.

File: blackbox/docs/todo.txt [hunk 8]
```diff
@@ -0,0 +1,354 @@
+admin/optimization.rst:.. highlight:: psql
```
Comment (mfussenegger) [inline blackbox/docs/todo.txt]: was this intentional checked in?

Comment (kevinkq) [reply blackbox/docs/todo.txt]: I will have to look at this together with @mechanomi.

File: blackbox/docs/admin/snapshots.rst (hunk 7 above)
Comment (kevinkq) [reply blackbox/docs/admin/snapshots.rst]: @kovrus I will have to look at this together with @mechanomi.

File: blackbox/docs/sql/statements/create-repository.rst (hunk 6 above)
Comment (kevinkq) [reply blackbox/docs/sql/statements/create-repository.rst]: @kovrus I will have to look at this together with @mechanomi.

File: blackbox/docs/config/cluster.rst (hunk 5 above)
Comment (kevinkq) [reply blackbox/docs/config/cluster.rst]: @kovrus I will have to look at this together with @mechanomi.

**Pull request closed without merging**